ios/FluentUI/Extensions/UIColor+Extensions.swift

```diff
@@ -98,6 +98,24 @@ extension UIColor {
             alpha: colorValue.a)
     }
 
+    /// Creates a `UIColor` instance with the specified three-channel, 8-bit-per-channel color value, usually in hex.
```

Comment:
"usually in hex"? That seems a bit imprecise, especially since we call the parameter "hexValue". What's the behavior if we pass in something something larger than the color space?

Reply:
(sorry my fluent reading of bit-shifting is quite rusty)

Reply:
So the only key is that the passed-in value must be 32 bits - this is enforced at build time by the integer being typed as a `UInt32`. Whether the integer is written as `0xE3008C` or as `14876812` or `0b111000110000000010001100` doesn't matter to us; they're all the same integer value.